test-files/io/OsmChangesetElementTest/ChangesetMergeExpected.osc

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<osmChange version="0.6" generator="Hootenanny">
+<osmChange version="0.6" generator="hootenanny">
```

Comment:
Lots of the changeset files have been updated to use the all lowercase `hootenanny` like all of the OSM file writers.

Reply:
Nice